src/libponyc/ast/lexer.c

```diff
@@ -607,7 +607,7 @@ static void normalise_string(lexer_t* lexer)
 
   // Trim trailing empty line
   size_t trim = 0;
-  for (size_t i = (lexer->buflen - 1); i>0; i--)
+  for (ssize_t i = (lexer->buflen - 1); i>0; i--)
```

Comment:
is there a known bug from this?

Reply:
not that i'm aware of.. just something that the undefined behavior sanitizer flagged (iirc)..

Reply:
https://github.com/ponylang/ponyc/pull/4359 created for this